cmd/wksctl/profile/enable/enable.go

```diff
@@ -60,8 +63,41 @@ func profileEnableArgs(cmd *cobra.Command, args []string) error {
 }
 
 func profileEnableRun(params profileEnableFlags) error {
-	repoURL := params.gitURL
 
+	const flexHelmOpAddonName = "flux-helm-op"
+	var flexHelmOpAddonPath = path.Join("base/addons", flexHelmOpAddonName)
```

Comment:
We should probably use `filepath` rather than `path` here...

Reply:
I've been using `path` a lot of the places. But yes it should be `filepath`. noted.
Thank you so much for your time reviewing this @jrryjcksn